test/metrics_number_dimension/resources/10_dimension/config.json

```diff
@@ -1,11 +1,12 @@
 {
   "agent": {
-    "metrics_collection_interval": 60,
+    "metrics_collection_interval": 10,
```

Comment:
Send to backend faster and more often, so less chance of timing error.

Reply:
Furthermore, my understanding is the more metrics you have the faster cw will index.